Packs/Sepio/README.md

```diff
@@ -0,0 +1,10 @@
+##### What does this pack do?
+- The HAC-1 pack provides visibility down to the physical layer, uncovering hidden or spoofed devices.
+- Interfaces supported - wired and wireless networks and USB peripherals.
+- Full asset visibility - each asset is identified and it's risk scored in compliance with Zero Trust guidelines.
+- Extensive reports and aleting embedded into the XSOAR automation process. 
+
+You may find the links below useful -
+- Sepio Systems Home page - https://sepio.systems
+- Sepio Systems HAC-1 listing in MSFT Azure Market place https://azuremarketplace.microsoft.com/en-us/marketplace/apps/sepiosystems.hac-1-oct20?tab=Overview
+- Sepio Systems Blog regarding XSOAR integration - https://www.paloaltonetworks.com/blog/security-operations/security-operations-sepiosystems-xsoar/

```

Comment:
Is markdown links looks better?
[Info about markdown links.](https://www.markdownguide.org/basic-syntax/#links)

```suggestion
###### You may find the links below useful -
- [Sepio Systems Home page.](https://sepio.systems)
- [Sepio Systems HAC-1 listing in MSFT Azure Market place.](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/sepiosystems.hac-1-oct20?tab=Overview)
- [Sepio Systems Blog regarding XSOAR integration.](https://www.paloaltonetworks.com/blog/security-operations/security-operations-sepiosystems-xsoar/)
```